country-info.csv

```diff
@@ -210,6 +210,7 @@ Texas A&M at Qatar,asia
 The Hong Kong Polytechnic University,asia
 Tilburg University,europe
 Tokyo Institute of Technology,asia
+TongJi University,asia

```

Comment:
The "J" should not be capitalized.

Reply:
Thank you for your remind and I have committed a new pull request which fixed the mismatch.

The entire eligible faculty of our lab (https://idvxlab.com/) has been added.

Reply:
It needs to be the entire faculty of the _CS department_, not just one lab.

Reply:
> Bump @Tsailgan - if you are planning to add the entire CS faculty, please do so. I'll close the PR if I don't hear back soon.

I will add the entire CS faculty as soon as possible.